src/Microsoft.DotNet.Build.Tasks.Feed/BuildManifest/WriteOrchestratedBuildManifestSummaryToFile.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.DotNet.VersionTools.BuildManifest.Model;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed.BuildManifest
+{
+    public class WriteOrchestratedBuildManifestSummaryToFile : Task
+    {
+        [Required]
+        public string File { get; set; }
+
+        [Required]
+        public string ManifestFile { get; set; }
+
+        public string SdkTableTemplateFile { get; set; }
+
+        public string RuntimeTableTemplateFile { get; set; }
+
+        public override bool Execute()
+        {
+            string contents = System.IO.File.ReadAllText(ManifestFile);
+            OrchestratedBuildModel model = OrchestratedBuildModel.Parse(XElement.Parse(contents));
+            EndpointModel blobFeed = model.Endpoints.First(e => e.IsOrchestratedBlobFeed);
+
+            string feedAssetsRoot = blobFeed.Url.Replace("/index.json", "/assets");
+
+            string sdkProductVersion = model.Builds.First(b => b.Name == "cli").BuildId;
+            string runtimeProductVersion = blobFeed.Artifacts.Blobs
+                .First(b => b.Id.StartsWith("Runtime/"))
+                .Id.Split('/')[1];
```

Comment:
This logic that assumes a lot about the Core-Setup paths will be removed once we shuttle product version directly through the manifest. (See https://github.com/dotnet/buildtools/pull/1871.)